ppr-ui/src/components/registration/RegistrationBar.vue

```diff
@@ -13,7 +13,7 @@
             <span class="pr-2"> Register a Manufactured Home</span>
           </v-btn>
           <registration-bar-type-ahead-list
-            v-else-if="hasRPPR"
+            v-else-if="hasRPPR && !isMhr"
```

Comment:
A fail-safe to fix some scenarios, where certain QS accounts would see PPR registration drop down in MHR Tab.